huggle/vandalnw.cpp

```diff
@@ -336,16 +336,18 @@ void VandalNw::ProcessGood(WikiEdit *edit, QString user)
 {
     edit->User->SetBadnessScore(edit->User->GetBadnessScore() - 200);
     QString sid = QString::number(edit->RevID);
-    this->Insert("<font color=blue>" + user + " saw a good edit on " + edit->GetSite()->Name + " to " + edit->Page->PageName + " by " + edit->User->Username
-                     + " (" + GenerateWikiDiffLink(sid, sid, edit->GetSite()) + ")" + "</font>", HAN::MessageType_User);
+    QStringList params;
+    params << user << edit->GetSite()->Name << edit->Page->PageName << edit->User->Username << GenerateWikiDiffLink(sid, sid, edit->GetSite());
+    this->Insert(_l("user-saw-good-edit", params), HAN::MessageType_User);
     MainWindow::HuggleMain->Queue1->DeleteByRevID(edit->RevID, edit->GetSite());
 }
 
 void VandalNw::ProcessRollback(WikiEdit *edit, QString user)
 {
     QString sid = QString::number(edit->RevID);
-    this->Insert("<font color=orange>" + user + " did a rollback on " + edit->GetSite()->Name + " of " + edit->Page->PageName + " by " + edit->User->Username
-                 + " (" + GenerateWikiDiffLink(sid, sid, edit->GetSite()) + ")" + "</font>", HAN::MessageType_User);
+    QStringList params;
```

Comment:
Yea, at first I inserted all five like normal but it didn't build. So I checked out if there's any other things with $3 to see how they do it :D